framework/src/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -788,6 +788,15 @@ public S copy() {
 
     @Override
     public S leastUpperBound(S other) {
+        return upperBound(other, true);
+    }
+
+    @Override
+    public S widenUpperBound(S other) {
+        return upperBound(other, false);
+    }
+
+    public S upperBound(S other, boolean lub) {
```

Comment:
A similar parameter below is called `shouldWiden`. It would be nice to use the same name and logic for the boolean parameter.